packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -25,12 +25,17 @@ const ProofOfOwnershipForm = ({ cards, updateAccountStatus }) => {
     });
     const [form_state, setFormState] = useStateCallback({ should_show_form: true });
     const form_ref = useRef();
+    const [document_upload_errors, setDocumentUploadErrors] = useState();
+    useEffect(() => {
+        setDocumentUploadErrors([]);
+    }, []);
     const fileReadErrorMessage = filename => {
         return localize('Unable to read file {{name}}', { name: filename });
     };
     const validateFields = values => {
+        const checked_indices = [];
         const errors = {};
-        errors.data = [];
+        errors.data = [...document_upload_errors];
```

Comment:
why dont you use document_upload_errors directly instead of passing its values in another array?

Reply:
@ali-hosseini-fs, it is because at some point when checking for file size and format, we modify specific indices of the array, which is not advised in react. Directly mutating state is bad.